Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -1742,12 +1742,26 @@ File-hiding is of course only the tip of the iceberg: you can accomplish a lot u
 
 ### References
 
-- Hacking Soft Tokens Paper by Bernhard Mueller - <https://packetstormsecurity.com/files/138504/HITB_Hacking_Soft_Tokens_v1.2.pdf>
-- OWASP MSTG Crackmes - <https://github.com/OWASP/owasp-mstg/tree/master/Crackmes>
+- Hacking Soft Tokens Paper - <https://packetstormsecurity.com/files/138504/HITB_Hacking_Soft_Tokens_v1.2.pdf>
```

Comment:
For papers please use the format as in the style guide 
https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md

Reply:
done, following: [shortname] [Author(s)], [Title] - Link

Reply:
👍